graphql/user/mutations.ts

```diff
@@ -28,6 +28,29 @@ class SupportMessage {
     subject: string;
 }
 
+@InputType()
+class AppFeedbackAttachment implements Attachment {
+    @Field()
+    Base64Content: string;
```

Comment:
limit to some reasonable size?

Reply:
Good catch.
I've added `2 * 1024 * 1024`... It's not exactly accurate because of the encoding, but I guess for a screenshot image it should be way more than enough.